var/spack/repos/builtin/packages/neo4j/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Neo4j(Package):
+    """Neo4j is the world’s leading Graph Database. It is a high performance
```

Comment:
```suggestion
    """Neo4j is the world's leading Graph Database. It is a high performance
```
Python 2 does not like Unicode